web-app/js/portal/cart/DownloadEstimator.js

```diff
@@ -30,6 +30,13 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
                 this._createDownloadEstimate(result, collection.getUuid(), callback);
             },
             failure: function(result, values) {
+                if (result.isTimeout) {
+                    var estimaterId = "downloadEst-" + collection.getUuid();
+                    var listOfElements = $('[id^='+estimaterId+']');
+                    if (listOfElements.length == 1) {
+                        listOfElements[0].innerHTML = '<div><span class="fa fa-fw fa-warning"></span>Unable to calculate download size.</div><div class="clear"></div>';
```

Comment:
This is a mix of jQuery and old school it seems.
It could be reduced to (untested):
`jQuery("#downloadEst-" + collection.getUuid()).html('<div><span class="fa fa-fw........')`